boards/common/particle-mesh/Makefile.dep

```diff
@@ -2,5 +2,9 @@ ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
 
+ifeq (1,$(PARTICLE_MONOFIRMWARE))
```

Comment:
Other boards do

```suggestion
ifneq (,$(filter stdio_cdc_acm,$(USEMODULE)))
```

 But turns out `usb_board_reset` does not depend on USB in any way, so your version is more correct.

Reply:
stdio_cdc_acm could have been enabled for various reasons, including someone using any other or no bootloader to flash this device.

The shipped implementation in reset.c is really bespoke to running a MONOFIRMWARE, to the extent that it may make sense to even guard it in reset.c (and see to it that #else there is an #error saying that so far usb reset is implemented for this board only if monofirmware is active, and otherwise please consult your bootloader manual to add it).